### ISSUE TITLE
[BUG] Window informing about update opens on every Steam page.

### ISSUE THREAD

**Issue by zandek83** [IsThereAnyDeal/AugmentedSteam#1953]
As title says. For a long time (maybe even half a year) a window informing about augmented steam update opens on every Steam page.
This window used to appear once a day, every day, so it wasn't that problematic, but now opening any Steam page causes this window to appear.
The option responsible for displaying / not displaying update information does not work.

### Specifications

  - Augmented Steam Version: 4.0.1
  - Browser Version:  126.0.6478.127

**Comment by CrackNButter** [comment]
same issue

**Comment by JohnDwaynsen** [comment]
Same here
Windows 11 Home 23H2 64 Bit
Chrome: Version 126.0.6478.127

**Comment by candela97** [comment]
Open the extension's service worker (for Chrome you need to enable developer mode, should be similar in Firefox), and in the console run `await chrome.storage.sync.get("version")`, is the result 4.0.1?

**Comment by JohnDwaynsen** [comment]
The result is 
{version: '2.1.0'}

**Comment by candela97** [comment]
Yes, unfortunately reinstalling might be the only way to fix this. The migration process that runs on update might have gotten stuck at some point, and we're looking into ways to improve it.

**Comment by zandek83** [comment]
I already tried this, problem came back next day.

**Comment by candela97** [comment]
@zandek83 There's something else going that's out of our hands then. Removing an extension is supposed to remove all data.

**Comment by tfedor** [comment]
@zandek83 I don't have much to go with here, but I suppose you don't see any errors in the browser console?
Also, do you write notes for games?

**Comment by tfedor** [comment]
Also, can you check, like last time with version, also this?
```ts
await chrome.storage.sync.get("excluded_version")
```

What does it return?

**Comment by tfedor** [comment]
And I found one last thing that I think might be breaking it that I would like to ask you to check.

Open Augmented Steam options, open dev tools (F12, or right click and Inspect) -> Application tab -> find IndexedDB -> Augmented Steam and expand that.

You should see the same thing as I see on the screenshot 
![image](https://github.com/user-attachments/assets/7728d1b9-e6b5-43f0-a01e-2ce7e9d85333)

Can you check that, and take a screenshot for me?

**Comment by zandek83** [comment]
> await chrome.storage.sync.get("excluded_stores")
I don't remember how to run the previous command, I can't run this command.

**Comment by zandek83** [comment]
Update.
After the last update (4.1.1) problem "solved itself" i.e. window with information about the update appears as before, i.e. once a day.

**Comment by candela97** [comment]
@zandek83  Can you try running this command https://github.com/IsThereAnyDeal/AugmentedSteam/issues/1953#issuecomment-2207195093 *before* opening any Steam page when starting a new day?

Also, out of curiosity, what happens if you disable syncing extension data in browser settings?
